tools/staramr/staramr_search.xml

```diff
@@ -89,11 +95,20 @@
                 </when>
                 <when value="none" />
             </conditional>
+            <conditional name="plasmidfinder_type">
```

Comment:
Same here. There's no need for `<conditional>`.

Reply:
Agreed, removed in my next commit